pkg/artifactory/resource_artifactory_saml_settings.go

```diff
@@ -31,6 +31,7 @@ type SamlSettings struct {
 	AutoRedirect              bool   `yaml:"autoRedirect" json:"autoRedirect"`
 	SyncGroups                bool   `yaml:"syncGroups" json:"syncGroups"`
 	VerifyAudienceRestriction bool   `yaml:"verifyAudienceRestriction" json:"verifyAudienceRestriction"`
+	UseEncryptedAssertion     bool   `yaml:"useEncryptedAssertion" json:"useEncryptedAssertion"`
```

Comment:
Purely my own ignorance. Why is `json` tag here? Does the endpoint `artifactory/api/saml/config` returns json or yaml?

Reply:
REST API call uses yaml.

Reply:
That's what I assumed. So why do we need the `json` tag?

Reply:
Seeing this usage in couple if places. Will test and remove them if all pass in next pass. I have merged this PR already.

Reply:
i see ... following for saml resource interactions.
PATCH call to api/system/configuration returns yaml response.
And, GET call to api/saml/config returns json response.

Reply:
Ah ok then.